docs/concepts/operator.md

```diff
@@ -154,12 +154,60 @@ The reconciler also manages a few observability-related components (more planned
 - `plutono`
 - `vali`
 - `prometheus-operator`
+- `alertmanager-garden` (read more [here](#observability))
 
 It is also mandatory to provide an IPv4 CIDR for the service network of the virtual cluster via `.spec.virtualCluster.networking.services`.
 This range is used by the API server to compute the cluster IPs of `Service`s.
 
 The controller maintains the `.status.lastOperation` which indicates the status of an operation.
 
+##### Observability
+
+By default, the `alertmanager-garden` deployed by `gardener-operator` does not come with any configuration.
+It is the responsibility of the human operators to design and provide it.
```

Comment:
Do we expect all operators to make use of alertmanager? Otherwise, it might make sense to make it configurable to let operators not deploy alertmanager or scale it to `0` replicas.
WDYT?

Reply:
Maybe yes. We could think about allowing to completely enable/disable the monitoring and logging stack, similar to how it's done in `gardenlet` ([ref](https://github.com/gardener/gardener/blob/master/example/20-componentconfig-gardenlet.yaml#L185)). I'm not sure whether just disabling `alertmanager` is a use case as well. For now, I'd prefer to keep it as until we learn that somebody wishes to disable this (or the entire stack). WDYT?

Reply:
Works for me. I was primarily wondering if there will be some gardener installations, which will just waste some resources due to alertmanager running without any configuration. However, you are completely right that this case could also be made for more components.
We can leave it as is for the time being.